### DISCUSSION TITLE
Error: Returned error: VM Exception while processing transaction: revert

### DISCUSSION THREAD

**Discussion by ghost** [ConsenSys-archive/truffle#5676] [General Q&A]
can someone help me i am unable to run my test file.....everything is compiling and migrating successfully but when i am running truffle test, terminal is throwing me this error

Artifacts written to C:\Users\HP\AppData\Local\Temp\test--8064-lEv7LhCGamYg
Compiled successfully using:

solc: 0.8.17+commit.8df45f5f.Emscripten.clang
Error: Returned error: VM Exception while processing transaction: revert
at module.exports (C:\Users\HP\Desktop\dappProject\migrations\2_deploy_contracts.js:19:14)
at processTicksAndRejections (node:internal/process/task_queues:96:5)
Truffle v5.6.4 (core: 5.6.4)
Node v16.16.0

here is my repo link
https://github.com/Dania001/Dapp-Project

please help me......

**Comment by cds-amal** [comment]
The `Transfer.transferReward`'s `require` statement reverts the transaction. The condition:`require(BalanceOf(msg.sender >= _value)` is not met. 
```
  BalanceOf[msg.sender]    : 100000000000000000000n
  _value                   : 1000000000000000000000000n
```

**Reply by ghost** [reply]
hey  @cds-amal i understood but how can i fix this....should i decrease the _value amount?
